main/src/ca/uwaterloo/flix/api/lsp/provider/HighlightProvider.scala

```diff
@@ -30,12 +30,16 @@ object HighlightProvider {
       case Some(entity) => entity match {
         case Entity.Case(caze) => highlightCase(uri, caze.sym)
 
+        case Entity.StructField(field) => throw new RuntimeException("JOE TODO")
```

Comment:
This is a bit dangerous because now VSCode crashes if you just hover. Ill merge for now, but it should be fixed fast.

Otherwise you could just return mkNotFound

Reply:
I can push all the changes at once

Reply:
I'll be quick

Reply:
No worries. Lets merge this and then you can make a new PR.